Mathlib/AlgebraicTopology/SimplexCategory/Defs.lean

```diff
@@ -167,6 +176,74 @@ noncomputable def inclusion.fullyFaithful (n : ℕ) :
 theorem Hom.ext {n} {a b : Truncated n} (f g : a ⟶ b) :
     f.toOrderHom = g.toOrderHom → f = g := SimplexCategory.Hom.ext _ _
 
+section Meta
+
+/-- Some quick attempts to prove that `⦋m⦌` is `n`-truncated (`⦋m⦌.len ≤ n`). -/
+macro "trunc" : tactic =>
+  `(tactic| first | decide | assumption |
+    dsimp only [SimplexCategory.len_mk]; omega)
```

Comment:
I remember this being discussed elsewhere, but can you remind me why `decide` and `assumption` are needed here?

Reply:
They are not needed (in that they don't solve any goals that aren't solved by `omega`), but I decided to keep them after some rudimentary benchmarking. Most of the goals solved by `trunc` can be solved by `decide` or `assumption`, and it is also nicer to carry around a single hypothesis rather than an omega proof.

That being said, I am happy to replace this with a simple `dsimp only [SimplexCategory.len_mk]; omega`.

Reply:
In fact, can you use `get_elem_tactic`? If that works well for your applications, that is probably the most robust with respect to future changes in tactics. But otherwise just `omega` is best, I think.

Reply:
`get_elem_tactic` was a perfect replacement for `trunc`, thanks!

Reply:
Actually, `get_elem_tactic` turned out to be slightly less effective than `omega` in some cases. I have restored the `trunc` tactic, but it just delegates to `omega` now.